doc, sqlite: add `--experimental-sqlite` note (#53907)
nodejs/node: doc/api/sqlite.md

added: v22.5.0
-->

> > Stability: 1.1 - Active development. Enable this API with the 

Reviewer: ```suggestion
> Stability: 1.1 - Active development. Enable this API with the
```

<!-- source_link=lib/sqlite.js -->

The `node:sqlite` module facilitates working with SQLite databases.

> This feature can only be used when the `--experimental-sqlite` CLI
> flag is specified. To access it:

Author: ```suggestion
To access this feature, use the `--experimental-sqlite` flag, and
then import it with:
```

I'm not sure which wording is better.

> flag is specified. To access it:

Reviewer: Can you move it to the `> Stability: 1.1 - Active development` block? Similarly to what we did in e.g. https://github.com/nodejs/node/blob/f232064f35df4e10ed8363863bca704fe878012f/doc/api/globals.md#L358-L359
